frontend/src/pages/Dashboard/Settings.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import Logout from '../../components/Logout/Logout';
 import styles from './page.module.css';
+import Table from '../../components/UserTables/DispatcherTable'
 
 const Settings = () => (
   <>
     <h1 className={styles.header}>Settings</h1>
     <div className={styles.pageContainer}>
       <Logout />
+      <Table />
```

Comment:
Could you import this as DispatcherTable instead of Table? It will make this easier to read since we have many other table components